UKHomeOffice/nmsw-ui
# NMSW-470 p7 Refactor declarationId - check your answers

# Ticket

NMSW-470

----

## AC

see https://github.com/UKHomeOffice/nmsw-ui/pull/269

----

## To test

- create a voyage up to task details page
- Click the check your answers link
- > the cya page should load with the declarationId in the url
- > it should show your GenDec FAL1 details
- Click the change link next to General declaration
- > it should load the GenDec upload page with the declarationId in the url
- Go back to CYA
- Click the change link next to crew
- > it should load the crew upload page with the declarationId in the url
- Go back to CYA
- Click the change link next to passengers
- > it should load the do you have passengers page with the declarationId in the url
- Go back to CYA
- Click the change link next to supporting documents
- > it should load the upload supporting documents page with the declarationId in the url
- Go back to CYA
- Click on the submit button
- > TESTING ONLY until we have a submit endpoint - it should console log the declarationId


----

## Notes
